src/equations/passive_tracers.jl

```diff
@@ -0,0 +1,222 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+@doc raw"""
+    PassiveTracerEquations(flow_equations)
+
+Adds passive tracers to the `flow_equations`. The tracers are advected by the flow velocity.
+These work for arbitrary NDIMS with arbitrary numbers of tracers NTracers. For one dimension, with 
+one tracer ``\xi`` and flow with density and velocity ``\rho, v`` respectively, the equation of the
+passive tracer is
+```math
+\frac{\partial \rho \xi}{\partial t} + \frac{\partial}{\partial x} \left( \rho v \xi \right) = 0
+```
+"""
+struct PassiveTracerEquations{NDIMS, NVARS, NTracers,
+                              FlowEquations <: AbstractEquations} <:
+       AbstractEquations{NDIMS, NVARS}
+    flow_equations::FlowEquations
+end
+
+function PassiveTracerEquations(flow_equations::AbstractEquations, n_tracers::Int)
```

Comment:
What about converting `n_tracers` to a keyword argument? Since the passive tracer equations do not carry an explicit dimension, one might be tempted/confused what to supply here.